sos/plugins/__init__.py

```diff
@@ -502,6 +502,35 @@ def do_cmd_output_sub(self, cmd, regexp, subst):
             replacements = None
         return replacements
 
+    def do_file_private_sub(self, pathregex):
+        '''Scrub certificate/key/etc information from files collected by sos.
+
+        Files matching the provided pathregex are searched for content that
+        resembles certificate, ssh keys, or similar information. Any matches
+        are replaced with "-----SCRUBBED $thing" where $thing is the specific
+        type of content being replaced, e.g. "-----SCRUBBED RSA PRIVATE KEY" so
+        that support representatives can at least be informed of what type of
+        content it was originally.
+
+        Positional arguments:
+            :param pathregex: A string or regex of a filename to match against
+        '''
+        certmatch = re.compile("-*BEGIN.*?-*END", re.DOTALL)
```

Comment:
cf. https://github.com/sosreport/sos/blob/master/sos/plugins/__init__.py#L497 - isn't it worth having standardised r.e. for this? (i.e. have the r.e. string stored in a constant, referenced from both places?)